packages/scaffold-ui/src/partials/w3m-connect-injected-widget/index.ts

```diff
@@ -54,7 +54,7 @@ export class W3mConnectInjectedWidget extends LitElement {
             return null
           }
 
-          if (!ConnectionController.checkInstalled(undefined, connector.chain)) {
+          if (!connector.info && !ConnectionController.checkInstalled(undefined, connector.chain)) {
```

Comment:
re:unit tests requests.

We don't have snapshot testing for UI components, we have E2E and unit tests. What we can do for this test something related with `ConnectorController`;

Not every connector have `info` field, only the injected ones. So we could add a test to the `ConnectorController.test.ts` like the following:

```js
// Example 
it('should throw error when adding injected connector without info field', () => {
    expect(() => ConnectorController.addConnector({
      ...injectedConnector,
      info: undefined
    })
    ).toThrow('Connector info is required for injected connectors')
  })
```

And we could update the `.addConnector` and `setConnectors` logics to check if the type is `INJECTED` and don't have `info` field, it should throw error.